admin_guide/manage_users.adoc

```diff
@@ -0,0 +1,135 @@
+
+=== Improper Deletion
+
+OpenShift user configuration is stored in several locations within OpenShift.
+Regardless of the identity provider, OpenShift internally stores details like
+role-based access control (RBAC) information and group membership. To completely
+remove user information, this data must be removed in addition to the user
+account.
+
+If an account is improperly deleted, the following type of error can be seen,
+even though the identification provider already shows the user *myuser* as
+existing:
+
+====
+----
+{
+    "MESSAGE": "E0412 06:07:48.142862     465 login.go:141] Unable to authenticate password: Error creating or updating mapping for: &api.DefaultUserIdentityInfo{ProviderName:\"htpasswd_auth\", ProviderUserName:\"myuser\", Extra:map[string]string{}} due to users \"myuser\" not found",
+    "PRIORITY": "6",
+    "SYSLOG_FACILITY": "3",
+    "SYSLOG_IDENTIFIER": "atomic-openshift-master",
+    "_BOOT_ID": "18e98783c33546b4a4346f6750ef84a5",
+    "_CAP_EFFECTIVE": "1fffffffff",
+    "_CMDLINE": "/usr/bin/openshift start master --config=/etc/origin/master/master-config.yaml --loglevel=2",
+    "_COMM": "openshift",
+    "_EXE": "/usr/bin/openshift",
+    "_GID": "0",
+    "_HOSTNAME": "master.example.com",
+    "_MACHINE_ID": "f9370ed252a14f73b014c1301a9b6d1b",
+    "_PID": "465",
+    "_SELINUX_CONTEXT": "system_u:system_r:init_t:s0",
+    "_SYSTEMD_CGROUP": "/system.slice/atomic-openshift-master.service",
+    "_SYSTEMD_SLICE": "system.slice",
+    "_SYSTEMD_UNIT": "atomic-openshift-master.service",
+    "_TRANSPORT": "stdout",
+    "_UID": "0",
+}
+----
+====
+
+This error is generated because OpenShift sees the user data in the internal
+database that does not correspond to the external identity.
+
+In OpenShift, two object types contain user data outside the identification
+provider: type user and type identity.
+
+Type user:
+
+====
+----
+$ oc get user ashley
+NAME      UID                                    FULL NAME   IDENTITIES
+peter     75e4b80c-dbf1-11e5-8dc6-0e81e52cc949               htpasswd_auth:ashley
+----
+====
+
+Type identity (from the user record):
+
+====
+----
+oc get identity htpasswd_auth:ashley
+NAME                  IDP NAME        IDP USER NAME   USER NAME   USER UID
+htpasswd_auth:ashley  htpasswd_auth   ashley          ashley      75e4b80c-dbf1-11e5-8dc6-0e81e52cc949
+----
+====
+
+Note the matching UID between the two object types. If you attempt to change the
+authentication provider after starting to use OpenShift, the user names that
+overlap will not work because of the entries in the identity list, which will
+still point to the old authentication method.
+
+=== Properly Deleting a User
+
+To delete a user, all three sources of the user must be removed:
+
+. Identification manager
+. User record
+. Identity record
+
+Once you complete these steps, a new account will be created in OpenShift
+when the user logs in again.
+
+==== Deleting the Identification Manager Entry
```

Comment:
Can we make these a procedure w/ numbered steps instead of individual subheadings per step?